web_src/css/explore.css

```diff
@@ -10,6 +10,28 @@
   margin-right: 5px;
 }
 
+.explore .filter .ui.right .menu {
+  max-height: 60vh;
+  overflow-y: auto;
+}
+
+.explore .filter .ui.dropdown {
+  float: right;
+}
+
+@media (max-width: 600px) {
+  .explore .filter {
+    flex-direction: column;
+  }
+  .explore .filter form {
+    min-width: 100%;
+  }
+  .explore .filter > div {
+    align-self: end;
+    margin-top: .25em;
+  }
```

Comment:
I see these `form` and `div` selectors likely to break during refactoring. Can you give them some unique classnames please?